components/HeaderNav/HeaderNav.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { StyleSheet, View, Text, Button } from 'react-native';
+
+export default function HeaderNav() {
+  return (
+    <View style={styles.nav}>
+      <View style={styles.navButtonContainer}>
+        <View style={styles.navButton}><Button title='change date'/></View>
```

Comment:
possibly also adding the 'filter by activity' button in this nav as well

Reply:
Oh yeah, we def will need that! I will implement that next.